src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -2766,6 +2830,86 @@ export async function getGenomicDataAsClinicalData(
     });
 }
 
+export function convertGenericAssayDataBinsToClinicalDataBins(
+    genericAssayDataBins: GenericAssayDataBin[]
+): ClinicalDataBin[] {
+    return genericAssayDataBins.map(gaDataBin => ({
+        attributeId: getGenericAssayChartUniqueKey(
+            gaDataBin.stableId,
+            gaDataBin.profileType
+        ),
+        count: gaDataBin.count,
+        end: gaDataBin.end,
+        specialValue: gaDataBin.specialValue,
+        start: gaDataBin.start,
+    }));
+}
+
+export async function getGenericAssayDataAsClinicalData(
```

Comment:
again, just wondering why we do this "conversion"

Reply:
We need clinical data for download and comparison feature. These methods/functions only support `ClincalData` objects. This method is for converting genomic data to clinical data and use those methods directly. I think it can also be implemented in a generic way

Reply:
Updated to use a generic type `DataBin`